apps/test/unit/templates/teacherDashboard/teacherSectionsReduxTest.js

```diff
@@ -557,6 +559,7 @@ describe('teacherSectionsRedux', () => {
       const stateWithSections = reducer(initialState, setSections(sections));
       assert.isNull(stateWithSections.sectionBeingEdited);
       const state = reducer(stateWithSections, beginEditingSection(12));
+      state.sectionBeingEdited.createdAt = undefined;
```

Comment:
same as above -- either `beginEditingNewSection` should be setting this, or you shouldn't be checking for the inclusion of `createdAt: undefined` below